app/assets/javascripts/modules/remove-filter.js

```diff
@@ -6,7 +6,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   GOVUK.Modules.RemoveFilter = function RemoveFilter () {
     this.start = function (element) {
-      $(element).on('click', '[data-module="remove-filter-link"]', toggleFilter)
+      element[0].addEventListener('click', function (e) {
+        var target = e.target
+        if (target.getAttribute('data-module') === 'remove-filter-link') {
```

Comment:
Nitpick: We can save a line here

```suggestion
        if (e.target.getAttribute('data-module') === 'remove-filter-link') {
```

Reply:
nice spot, was being too literal, fixed :)